### ISSUE TITLE
Remove borsh

### ISSUE THREAD

**Issue by kevinheavey** [solana-program/config#11]
borsh is slow to compile and contributes to dependency hell in a big way. Nothing will break if we remove it from the client now

**Comment by wowinter13** [comment]
I suppose this one could be closed? It looks like it was addressed.

@buffalojoec ?

**Comment by buffalojoec** [comment]
> I suppose this one could be closed? It looks like it was addressed.
> 
> [@buffalojoec](https://github.com/buffalojoec) ?

Nope, unfortunately: https://github.com/solana-program/config/blob/3d8e19a01479e24e040f0fcb4e16f72ed4f7a16a/clients/rust/Cargo.toml#L16

**Comment by wowinter13** [comment]
Ah sorry
I checked `program/Cargo.toml`
